components/Grid/GridColumn.vue

```diff
@@ -360,7 +343,7 @@ export default {
         },
         getColumnFixedIndex() {
             if (this.$el.style.transform) {
-                const xTransform = this.$el.style.transform.replace(/[^0-9\-.,]/g, '');
+                const xTransform = this.getElementTransform();
```

Comment:
Always when you call this function you convert value to number. Convert values in the middle of the function.
I think using 'Unary Operator' is shorter but we lacks readability.